doc/manuals/sprokit/embedded_pipeline.rst

```diff
@@ -0,0 +1,327 @@
+Embedded Pipeline
+=================
+
+The embedded pipeline class provides a way to instantiate and run a
+sprokit pipeline within a program. The main use case for an embedded
+pipeline is in cases where a program calculates or obtains the input
+and needs the output for local processing. A prime example would be a
+GUI that runs a pipeline and display the results.
+
+The pipeline to be run is passed to the ``build_pipeline()`` method as a
+stream. The stream can be from a file or from a string built at run
+time. If the pipeline is coming from a file, it is useful to supply
+the directory portion (up to but not including the last '/') to the
+call so the error messages can provide the real file name and any
+*relativepath* modifiers can supply an accurate prefix. If the
```

Comment:
`*relativepath*`... is that correct?

Reply:
I am... surprised it is not `relative_path` or `relativePath`... but yes, also being emphasized rather than literal seems odd too.